docs/pages/reference/access-controls/authentication.mdx

```diff
@@ -267,3 +267,50 @@ See [GitHub OAuth 2.0](../../admin-guides/access-controls/sso/github-sso.mdx) fo
 
 </TabItem>
 </Tabs>
+
+## Require Displaying Message of the Day (MOTD)
```

Comment:
```suggestion
## Require displaying a message of the day
```
Suggestions to:
- Reduce the length of the heading
- Use sentence case, which we require in our style guide